golem_messages/message/tasks.py

```diff
@@ -330,33 +329,20 @@ def subtask_id(self):
     def task_to_compute(self):
         return self
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        if not hasattr(self, '_ethsig'):
-            self._ethsig = None
-
     def serialize(self, *args, **kwargs):  # pylint: disable=arguments-differ
         serialized = super().serialize(*args, **kwargs)
         sig_length = self.ETHSIG_LENGTH - 1
-        if self._ethsig and len(self._ethsig) != sig_length:
+        if self.ethsig and len(self.ethsig) != sig_length:
```

Comment:
@rwrzesien I don't think this is needed here -> this field is now serialized normally, alongside the other ones, so the length check isn't needed - if the signature is wrong - it just won't pass `verify_ethsig()` ...

Reply:
:+1: done